libGraphite/quickdraw/pict.cpp

```diff
@@ -448,6 +437,14 @@ auto graphite::qd::pict::parse(graphite::data::reader& pict_reader) -> void
                 read_long_comment(pict_reader);
                 break;
             }
+            case opcode::short_comment: {
+                pict_reader.move(2);
```

Comment:
Might be an idea to copy the style of the `read_long_comment` above, and to actually skip over the contents of the comment if they exist.

Reply:
As best I could tell from the samples I had and from the imagemagick code, the size is always 2. Does seem like a pretty short comment... 🤔